docs/pages/docs/index.mdx

```diff
@@ -9,22 +9,22 @@ import Features from "../../components/Features";
 
 <Features page="docs" detailed={false} />
 
-Turborepo is a blazing fast build system for JavaScript/TypeScript _monorepos_: codebases containing multiple
-projects, often using multiple frameworks, in a single unified
+Turborepo is a build system for JavaScript/TypeScript _monorepos_: codebases containing multiple
```

Comment:
"blazing fast" rubs me the wrong way 😆; it's unnecessary tooting of our own horn. I prefer that we're humble and show results instead (but that would be out of scope here). Also, what does "fast" mean?? Every single tool probably says they're the easiest and fastest to use!

Reply:
"blazing fast" is a keyword that can enable a brand over just a tool

See: https://vercel.com/#:~:text=Vercel%20combines%20the%20best%20developer%20experience%20with%20an%20obsessive%20focus%20on%20end%2Duser%20performance.%0AOur%20platform%20enables%20frontend%20teams%20to%20do%20their%20best%20work.

Reply:
We shouldn't take the tone of the front page descriptions lightly

Reply:
https://vercel.com/docs#:~:text=We%20provide%20a,is%20the%20default.

Reply:
> We shouldn't take the tone of the front page descriptions lightly

that's where I'm coming from too :)

> communicating brand e and values

💯. To me "blazing fast" communicates hype / trendy tech, and I wanted to communicate the real, underlying value.

Regardless! Since you've both pointed it out and [the original thesis][1] for the PR is to _not_ include anything that needs more conversation, I'll take Tom's suggestion and revert this :)

[1]: https://github.com/vercel/turborepo/pull/1701#issue-1339739892